apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -50,6 +50,51 @@
 {{#inline}}
 </span>
 {{/inline}}
+{{/radio}}
+{{^radio}}
+<!-- Weirdly enough putting this in a div causes it to display without the inline -->
+<span class="py-1 d-inline-flex align-items-center">
+    <script>
+        $(() => {
+            $('#pl-multiple-choice-select-{{uuid}}').selectpicker({})
+                .on('rendered.bs.select loaded.bs.select shown.bs.select', (e) => {
+                    // If any of the options has math, render the math as needed
+                    // Events: loaded (when the page opens), rendered (when the value changes),
+                    // shown (when the dropdown extends)
+                    MathJax.typesetPromise();
+                });
+        });
+    </script>
+    <select name="{{name}}" data-style="btn-outline-secondary"
+            id="pl-multiple-choice-select-{{uuid}}" {{^editable}}disabled{{/editable}}>
+        <option disabled selected value></option>
```

Comment:
I don't love how this gives us an unclickable empty space in the dropdown:

<img width="236" alt="Screenshot 2024-02-07 at 11 34 52" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/f492c052-0c61-48b8-af2e-0b4d9f0e42c1">

The docs suggest that we can use `title` to improve this, maybe there's another way too?

Reply:
I thought it was a little weird too, but the disabled non-selectable option makes the semantics of the dropdown match that of the radio button more closely (like the radio button, there's no option selected by default). Where in the docs is `title` described? I can play around with it a little bit.

Reply:
Just found the docs and I agree, it looks cleaner doing that while still achieving the same behavior. Will change 👍🏽

Reply:
Changed here: https://github.com/PrairieLearn/PrairieLearn/pull/8903/commits/6013f0316e4186e130eedaa539a47ddc0e03d97b

Reply:
Nice, I like that much better!